src/resolve/resolvconf-compat.c

```diff
@@ -34,13 +34,14 @@ static int resolvconf_help(void) {
                "     --version  Show package version\n"
                "  -a            Register per-interface DNS server and domain data\n"
                "  -d            Unregister per-interface DNS server and domain data\n"
+               "  -p            Make the interface not used as the default route\n"
```

Comment:
hmm, the sentence confuses me.

maybe:

"Do not use this interface as default route"?